pkg/server/events.go

```diff
@@ -198,6 +198,11 @@ func (em *eventMonitor) handleEvent(any interface{}) error {
 		e := any.(*eventtypes.TaskExit)
 		logrus.Infof("TaskExit event %+v", e)
 		// Use ID instead of ContainerID to rule out TaskExit event for exec.
+		if e.ID == "" {
```

Comment:
Hm, I think current logic can handle this case, and firecracker should have ID here.

Reply:
It would handle it, but is that a requirement that we should force on all containers created in containerd? 

Would be nice to have a namespace filter on these events I think.

Reply:
IMO the behavior should be equivalent to does not exist, vs error prefix is invalid..

Reply:
Let's at least keep this PR only add the namespace filter for easy cherrypick

Reply:
ok.. the filter for namespace will be enough..

Reply:
sigh.. so the subscriptions return any events that match, so we could request any for our namespace or any for the ones we already subscribe to, but it does not filter out by namespace unless that is the only filter used.